src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -15,98 +15,114 @@
+      (get-in contact [:names :three-words-name])))
 
-    "Dummy default title"))
+(defmulti notification-component :type)
 
-(defn activity-icon
-  [{:keys [type]}]
-  (case type
-    constants/activity-center-notification-type-contact-request
-    :main-icons2/add-user
-    :main-icons2/placeholder))
+;;;; Contact request notifications
 
-(defn activity-context
-  [{:keys [message last-message type]}]
-  (case type
-    constants/activity-center-notification-type-contact-request
-    (let [message     (or message last-message)
-          contact     (<sub [:contacts/contact-by-identity (:from message)])
-          sender-name (or (get-in contact [:names :nickname])
-                          (get-in contact [:names :three-words-name]))]
-      [[context-tags/user-avatar-tag
-        {:color          :purple
-         :override-theme :dark
-         :size           :small
-         :style          {:background-color colors/white-opa-10}
-         :text-style     {:color colors/white}}
-        sender-name
-        (multiaccounts/displayed-photo contact)]
-       [rn/text {:style {:color colors/white}}
-        (i18n/label :t/contact-request-sent)]])
-    nil))
+(defmethod notification-component constants/activity-center-notification-type-contact-request
+  [{:keys [id] :as notification}]
+  (let [message   (or (:message notification) (:last-message notification))
+        contact   (<sub [:contacts/contact-by-identity (:author notification)])
+        pressable (case (:contact-request-state message)
+                    constants/contact-request-message-state-accepted
+                    ;; NOTE [2022-09-21]: We need to dispatch to
+                    ;; `:contact.ui/send-message-pressed` instead of
+                    ;; `:chat.ui/navigate-to-chat`, otherwise the chat screen looks completely
+                    ;; broken if it has never been opened before for the accepted contact.
+                    [animation/pressable {:on-press (fn []
+                                                      (>evt [:hide-popover])
+                                                      (>evt [:contact.ui/send-message-pressed {:public-key (:author notification)}]))}]
+                    [:<>])]
+    (conj pressable
+          [activity-logs/activity-log
+           (merge {:title     (i18n/label :t/contact-request)
+                   :icon      :main-icons2/add-user
+                   :timestamp (datetime/timestamp->relative (:timestamp notification))
+                   :unread?   (not (:read notification))
+                   :context   [[context-tags/user-avatar-tag
+                                {:color          :purple
+                                 :override-theme :dark
+                                 :size           :small
+                                 :style          {:background-color colors/white-opa-10}
+                                 :text-style     {:color colors/white}}
+                                (sender-name contact)
```

Comment:
I don't see any `colors/theme-colors` usage. Just checking if you tested the UI in dark mode.

Reply:
The AC is supposed to always be viewed as "if in dark mode", contrary to other parts of the app that adapt to light/dark. But @OmarBasem, there's an outstanding issue after some recent changes in `develop` that made the AC look bad in light mode, but @smohamedjavid and I prefer to fix this in a separate issue if that's okay by you.

Reply:
Of course it's okay by me :)